cpp/include/cuml/fil/fil.h

```diff
@@ -201,9 +201,11 @@ void free(const cumlHandle& h, forest_t f);
  *  @param data array of size n * cols (cols is the number of columns
  *      for the forest f) from which to predict
  *  @param num_rows number of data rows
+ *  @param for classifier models, predict_proba forces to output both class probabilities
+ *      instead of binary class prediction. format matches scikit-learn API
  */
 void predict(const cumlHandle& h, forest_t f, float* preds, const float* data,
-             size_t num_rows);
+             size_t num_rows, bool predict_proba = false);
```

Comment:
In the doxygen for this method, may be it's worth mentioning that the size of `preds` array when `predict_proba == true` should be `nClasses * num_rows`?